staff/felipe-fernandez/blog-layout/index.html

```diff
@@ -0,0 +1,87 @@
+               
+            </article>
+
+            <article>
+                <h2>
+                Title  Article 2
+                </h2>
+                <div class="fakeimg">Image</div>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+            </article>
+            <article>
+                <h2>
+                Title  Article 3
+                </h2>
+                <div class="fakeimg">Image</div>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+            </article>
+            <article>
+                <h2>
+                Title  Article 4
+                </h2>
+                <div class="fakeimg">Image</div>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+            </article>
+        </section>
+
+        <section class="aside">
+            <aside>
+                <h2>About Me</h2>
+                <div class="fakeimg">Image</div>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
+            </aside>
+            <aside>
+                <h2>Popular post</h2>
+                <div class="fakeimg">Image</div>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
+            </aside>
+            <aside>
+                <h2>Follow me</h2>
+                <div class="fakeimg">Image</div>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
+            </aside>
```

Comment:
Aside elements brake when rezising window!

Reply:
Fixed! I've rebuild the layout with flex. Thanks!